src/CheckoutSdk/Sessions/ChallengeCancelReason.cs

```diff
@@ -7,15 +7,17 @@ public enum ChallengeCancelReason
         [EnumMember(Value = "cardholder_cancel")]
         CardholderCancel,
 
-        [EnumMember(Value = "transaction_timed_out")]
-        TransactionTimedOut,
-
         [EnumMember(Value = "challenge_timed_out")]
         ChallengeTimedOut,
+        
+        [EnumMember(Value = "sdk_timed_out")] SdkTimedOut,
```

Comment:
Can you put this on two lines like the other entries?

Reply:
I'm really not sure what the difference is between SDK timed out and transaction timed out

Reply:
👍🏼